src/components/card/card.stories.mdx

```diff
@@ -399,16 +398,16 @@ Click on the card to see the effect
             justifyContent="space-around"
           >
             <Box flexGrow={1}>
-              <IconButton>
+              <IconButton onAction={() => {}}>
```

Comment:
I think there will be accessibility warnings thrown here, as you're updating this anyway can you add aria-labels to these `IconButton` as well please?